pkgs/tools/misc/online-judge-tools/default.nix

```diff
@@ -0,0 +1,42 @@
+{ git, lib, python3Packages, ... }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "online-judge-tools";
+  version = "11.5.1";
+  format = "wheel";
+
```

Comment:
if this is going to be packaged as a module. then all you need to do is `  online-judge-tools = with python3Packages; toPythonApplication online-judge-tools;` in `all-packages.nix`